frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -278,6 +278,14 @@ export const menuActionDeleteVMI = (kindObj: K8sKind, vmi: VMIKind): KebabOption
   accessReview: asAccessReview(kindObj, vmi, 'delete'),
 });
 
+export const menuActionOpenConsole = (kindObj: K8sKind, vmi: VMIKind): KebabOption => ({
+  label: `Open Console`,
+  callback: () =>
```

Comment:
can we do the new tab here instead?

Reply:
it does open a new tab, if not its a user browser preference.

Reply:
I see, shouldn't we rather always open it a new tab to be consistent?